pkg/apis/contrail/v1alpha1/zookeeper_types.go

```diff
@@ -42,9 +42,6 @@ type ZookeeperConfiguration struct {
 // ZookeeperStatus defines the status of the zookeeper object.
 // +k8s:openapi-gen=true
 type ZookeeperStatus struct {
-	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
-	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
-	// Add custom validation using kubebuilder tags: https://book.kubebuilder.io/beyond_basics/generating_crd.html
 	Active *bool                `json:"active,omitempty"`
```

Comment:
May be we can add universal "status" field here like we have added in other resources.

Reply:
i'll do it in the next pr if you don't mind